openapis/swagger.yaml

```
@@ -1975,40 +1996,85 @@ components:
     UserInfo:
       type: object
       required:
-        - pdc_id
+        - user_id
+        - sub
         - username
-        - resources_granted
+        - name
+        - display_name
+        - preferred_username
+        - phone_number
+        - email
+        - is_admin
+        - role
         - project_access
         - certificates_uploaded
-        - email
+        - resources_granted
+        - groups
         - message
-        - primary_google_service_account
       properties:
-        pdc_id:
+        user_id:
+          type: number
+          description: 'This value is deprecated in favor of sub.'
+        sub:
           type: string
+          description: 'Subject Identifier. A locally unique and never reassigned identifier within the Issuer for the end-user.'
         username:
+          type: string
+          description: 'This value is deprecated in favor of name.'
+        name:
+          type: string
+          description: 'The full name of the end-user.'
+        display_name:
+          type: string
+          description: 'The display name of the end-user.'
+        preferred_username:
+          type: string
+          description: 'The preferred username of the end-user.'
+        phone_number:
+          type: string
+          description: 'The phone number of the end-user.'
+        email:
+          type: string
+          description: 'The email of the end-user'
+        is_admin:
+          type: boolean
+          description: 'Boolean value stating if end-user is an admin or not'
+        role:
           type: string
           description: ''
-        resources_granted:
+        project_access:
+          type: object
+          description: ''
```

```suggestion
          description: 'This value is deprecated in favor of authz and resources.'
```
or `list of projects the user has access to` if we don't want to say it's deprecated